develop/develop-images/build_enhancements.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Build images with BuildKit
+description: Learn the new features of Docker Build with BuildKit
+keywords: build, security, engine, secret, BuildKit
+---
```

Comment:
This page has been removed in https://github.com/docker/docs/pull/15958, bad rebase?

Reply:
Oops yeah. I thought I fixed this one...